src/YoTennis/Models/State.cs

```diff
@@ -7,17 +7,17 @@ namespace YoTennis.Models
 {
     public class State
     {
-        public PlayerScore ScoreOnSets { get; set; }
-        public List<Set> ScoreInSets { get; set; }
-        public Game ScoreInGame { get; set; }
+        public Score SetScore { get; set; }
+        public List<Set> Sets { get; set; }
+        public Game GameScore { get; set; }
```

Comment:
`GameScore` is not a `Score`. It's a `Game`. So, why did you call it `*Score`? I suggest to call it just `Game`.